src/extension.ts

```diff
@@ -340,9 +340,11 @@ export async function activate(extCtx: ExtensionContext): Promise<void> {
 
     extension = new Extension(extCtx)
     await extension.activate()
-    await appendWatcherExclude(['.flatpak'])
+
+    await appendWatcherExclude(['.flatpak/**', '_build/**'])
```

Comment:
we have the _build in const, can we use that here?

Reply:
Maybe it should be integration specific too?

Reply:
That would be good i guess yes, but then the integrations would need to provide the build commands somehow because both are related. I wouldn't spend too much time on that for now but it is an interesting thing to investigate in the future

Reply:
I think the Extension integrations shouldn't interfere directly with the build pipeline. Maybe it would have its own provider like some Build System integration